liberapay/exceptions.py

```diff
@@ -163,8 +163,8 @@ class TooManyLoginEmails(LazyResponseXXX):
     code = 429
     def msg(self, _):
         return _(
-            "You have consumed your quota of email logins, please try again tomorrow, "
-            "or contact support@liberapay.com."
+            "You have exceeded your quota of email logins, please try again tomorrow, "
+            "or contact support@liberapay.com"
```

Comment:
same as above

Reply:
Why did you remove the full stop?

Reply:
Makes it easier to copy the e-mail address. It is the small things :)

Reply:
I would prefer to make it clickable, it's both grammatically correct and convenient.